tests/Behat/ContentLockContext.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_editorial\Behat;
+
+use Drupal\DrupalExtension\Context\RawDrupalContext;
+
+/**
+ * Defines step definitions for testing the Content Lock feature.
+ */
+class ContentLockContext extends RawDrupalContext {
+
+  /**
+   * Install the content_lock component.
+   *
+   * @BeforeScenario @content_lock
+   */
+  public function enableContentLock() {
+    // Enable the Content Lock feature.
+    \Drupal::service('module_installer')->install(['oe_editorial_content_lock']);
+  }
+
+  /**
+   * Uninstall the content_lock component.
+   *
+   * @AfterScenario @content_lock
+   */
+  public function disableContentLock() {
+    $modules = [
+      'oe_editorial_content_lock',
+      'content_lock',
```

Comment:
why are we including in dependencies module content_lock but after finishing tests uninstalling this module?

Reply:
When we enable oe_content_lock on the enableContentLock function this enables content_lock as its a dependency. When we cleanup once the scenario is finished, both modules need to be disabled.